.gitignore

```diff
@@ -1 +1,41 @@
+# Clang code formatting file
```

Comment:
This helps me on both linux and mac... it is almost entirely binaries which shouldn't be committed anyway.